packages/site/src/layout/AnimatedPresenceDisclosure.tsx

```diff
@@ -1,29 +1,66 @@
 import { AnimatedPresence, Button, Typography } from "@jobber/components";
-import React, { useState } from "react";
-import { Link } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import { Link, useLocation } from "react-router-dom";
 import styles from "./NavMenu.module.css";
 
 interface AnimatedPresenceDisclosureProps {
   readonly children: React.ReactNode;
   readonly title: React.ReactNode;
   readonly to: string;
+  readonly selected?: boolean;
 }
 
 function AnimatedPresenceDisclosure({
   children,
   title,
   to,
+  selected,
 }: AnimatedPresenceDisclosureProps) {
-  const [isOpen, setIsOpen] = useState(false);
+  const location = useLocation();
```

Comment:
I should have done this in my changes to `ContentLoader` as well but if we are just using pathname it would better to just explicitly grab the pathname like `const { pathname } = useLocation();`. Not a big deal but the benefit here is that if parts of the location other than pathname change it won't cause a re-render.

Reply:
Great call out - adjusted here 7f6e6b9d2648a0bde923bd53e5e89529b48f1d1b